connectors/concur/src/main/java/com/vmware/connectors/concur/ConcurController.java

```diff
@@ -86,6 +86,17 @@ public ConcurController(WebClient rest,
         this.concurrRequestTemplate = concurRequestTemplate;
     }
 
+    @GetMapping("/test-auth")
+    public Mono<ResponseEntity<Void>> verifyAuth(@RequestHeader(name = AUTHORIZATION_HEADER) final String authHeader,
+                                 @RequestHeader(name = BACKEND_BASE_URL_HEADER) final String baseUrl) {
+        return this.rest.head()
+                .uri(baseUrl)
```

Comment:
Did you test this?

Reply:
Yes. Forgot to mention when I raise the PR. I have updated the ticket with the curl command used for testing the concur test auth.

https://jira-euc.eng.vmware.com/jira/browse/APF-1399?focusedCommentId=1042009&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1042009